src/Controls/src/Core/Platform/iOS/Extensions/TextExtensions.cs

```diff
@@ -45,9 +45,11 @@ public static void UpdateText(this UITextView textView, InputView inputView)
 			var cursorPosition = textView.IsFirstResponder ? textView.GetCursorPosition(cursorOffset) : newText.Length;
 
 			if (oldText != newText)
+			{
 				textView.Text = newText;
 
-			textView.SetTextRange(cursorPosition, 0);
+				textView.SetTextRange(cursorPosition, 0);
```

Comment:
Can we consolidate these two `UpdateText` methods?

Reply:
not sure, they are different native types - text VIEW vs text FIELD

Reply:
Yea, I think you'd just have to use an Action<PlatformView, string> for the settext and updatetext 

or just use a switch statement inside UpdateText since there's not going to be a lot of reuse beyond those two types

Reply:
Done!